### ISSUE TITLE
[ BUG ] `Import-FalconConfig` ignores `FirewallGroup` 

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#380]
**Describe the bug**
When attempting to use `Import-FalconConfig` with an archive that has one or more `FirewallGroup` items, it is ignored

**To Reproduce**
Create an export that includes `FirewallGroup` (either by itself, or with other items like `FirewallPolicy`) and attempt to import

**Expected behavior**
Successful `FirewallGroup` creation

**Environment (please complete the following information):**
 - OS: Windows 11, macOS
 - PowerShell: 5.1, 7.4
 - PSFalcon: 2.2.6

**Comment by bk-cs** [comment]
This issue will be fixed in the next release. If you'd like to resolve it for your local module before release, you can replace `public\psf-config.ps1` using the steps outlined below.

```powershell
Import-Module -Name PSFalcon
$ModulePath = (Show-FalconModule).ModulePath
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/dev/public/psf-config.ps1 -UseBasicParsing).Content > (Join-Path (Join-Path $ModulePath public) psf-config.ps1)
```

**Comment by bk-cs** [comment]
This fix is included in the [2.2.7 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.2.7).
